parts/plugins/x-apache.py

```diff
@@ -188,7 +188,7 @@ def build(self):
 
         self.run(
             ['make', '-j{}'.format(
-                snapcraft.common.get_parallel_build_count())],
+                self.project.parallel_build_count)],
```

Comment:
You're right. It didn't come up since I've only tested with the PHP branch which don't have 3rd party modules...